wazo_acceptance/steps/user.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import random
+import string
+
+from behave import given
+
+
+def random_string(length, sample=string.ascii_lowercase):
+    return ''.join(random.choice(sample) for _ in range(length))
+
+
+@given('there is a user')
+def given_there_is_a_user(context):
+    context.username = random_string(10)
+    context.password = random_string(10, sample=string.printable)
+
+    body = {
+        'firstname': random_string(10),
+        'username': context.username,
+        'password': context.password,
+    }
+
+    user = context.auth_client.users.new(**body)
+
+    context.add_cleanup(context.auth_client.users.delete, user['uuid'])
```

Comment:
If the same logic become used elsewhere, we will extract it into helper?

Reply:
Yeah I did not want to choose what to put in the user_helper module :grinning: 

I guess we will see what happens after a few steps using users